extension/content.css

```diff
@@ -82,8 +82,17 @@
 }
 
 /* remove commentbox toolbar */
-.toolbar-commenting {
-	display: none !important;
+.toolbar-group {
+    display: none;
+}
+.toolbar-group:last-child {
+    display: inline-block;
+}
+.toolbar-group .toolbar-item {
+    display: none;
+}
+.toolbar-group .js-saved-reply-container {
+	display: block;
```

Comment:
If you don't need `!important` shouldn't it be omitted? It's very heavy handed and would make it hard for this extension to be compatible with other extensions. They have to be super selective _and_use `!important`. It'll even override element styles you add via the inspector when you debug or play with stuff.

Personally I try to avoid using it unless really necessary, but can add it if there's a consensus to do that.